src/main/java/frc/robot/commands/RunWinchUntilHeight.java

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.Climber;
+
+public class RunWinchUntilHeight extends Command {
+  /** Creates a new RunWinchUntil. */
+  Climber m_climber;
+  double m_wantedPosition;
+
+  public RunWinchUntilHeight(Climber climber, double wantedPosition) {
+    // Use addRequirements() here to declare subsystem dependencies.
+    m_climber = climber;
+    m_wantedPosition = wantedPosition;
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    m_climber.run(1, 1);
+
```

Comment:
This does not work properly if the winches are at different heights, which will happen. Compute each speed separately.

Reply:
This means setting the speed in execute().